### ISSUE TITLE
Serving time increase at each compilation

### ISSUE THREAD

**Issue by sebsheep** [wking-io/elm-live#166]
The command line im using is : 
`elm-live src/Main.elm -- --output=main.js`

I have an `index.html` which links `main.js`, and my browser is on this `index.html`. I'm running this command under Debian 4.9.65-3+deb9u2, my version of elm-live is elm-live v3.0.1.

Each time I save my file, the serving time increase (even if there is in fact no change in my code except a new blank line). The following logs show the phonomenon (each save is  only a new blank line, which is automatically reformated by elm-format).

Compilation is always almost instantaneous. If I keep going, the serving time of "/main.js" can increase until 4s ; and maybe more, but at this point, I restart elm-live.

The problem is solved by restarting elm-live.

```
elm-live:
  You’ve changed `src/Main.elm`. Rebuilding!

Success! Compiled 1 module.                                          
[0143] 80ms          0B GET    304 /
[0143] 66ms       407KB GET    200 /main.js
[0143] 0ms           0B GET    304 /img/bulle-reversed.svg

elm-live:
  You’ve changed `src/Main.elm`. Rebuilding!

Success! Compiled 1 module.                                          
[0146] 10ms          0B GET    304 /
[0146] 163ms      407KB GET    200 /main.js
[0147] 1ms           0B GET    304 /img/bulle-reversed.svg

elm-live:
  You’ve changed `src/Main.elm`. Rebuilding!

Success! Compiled 1 module.                                          
[0149] 15ms          0B GET    304 /
[0149] 241ms      407KB GET    200 /main.js
[0150] 25ms          0B GET    304 /img/bulle-reversed.svg

elm-live:
  You’ve changed `src/Main.elm`. Rebuilding!

Success! Compiled 1 module.                                          
[0152] 61ms          0B GET    304 /
[0153] 86ms       407KB GET    200 /main.js
[0153] 0ms           0B GET    304 /img/bulle-reversed.svg

elm-live:
  You’ve changed `src/Main.elm`. Rebuilding!

Success! Compiled 1 module.                                          
[0157] 53ms          0B GET    304 /
[0157] 218ms      407KB GET    200 /main.js
[0158] 64ms          0B GET    304 /img/bulle-reversed.svg

elm-live:
  You’ve changed `src/Main.elm`. Rebuilding!

Success! Compiled 1 module.                                          
[0160] 64ms          0B GET    304 /
[0161] 309ms      407KB GET    200 /main.js
[0161] 5ms           0B GET    304 /img/bulle-reversed.svg

elm-live:
  You’ve changed `src/Main.elm`. Rebuilding!

Success! Compiled 1 module.                                          
[0164] 60ms          0B GET    304 /
[0164] 150ms      407KB GET    200 /main.js
[0165] 1ms           0B GET    304 /img/bulle-reversed.svg

elm-live:
  You’ve changed `src/Main.elm`. Rebuilding!

Success! Compiled 1 module.                                          
[0167] 96ms          0B GET    304 /
[0168] 422ms      407KB GET    200 /main.js
[0168] 5ms           0B GET    304 /img/bulle-reversed.svg

elm-live:
  You’ve changed `src/Main.elm`. Rebuilding!

Success! Compiled 1 module.                                          
[0170] 131ms         0B GET    304 /
[0172] 455ms      407KB GET    200 /main.js
[0172] 1ms           0B GET    304 /img/bulle-reversed.svg

elm-live:
  You’ve changed `src/Main.elm`. Rebuilding!

Success! Compiled 1 module.                                          
[0174] 120ms         0B GET    304 /
[0175] 480ms      407KB GET    200 /main.js
[0176] 1ms           0B GET    304 /img/bulle-reversed.svg

elm-live:
  You’ve changed `src/Main.elm`. Rebuilding!

Success! Compiled 1 module.                                          
[0178] 99ms          0B GET    304 /
[0179] 246ms      407KB GET    200 /main.js
[0179] 1ms           0B GET    304 /img/bulle-reversed.svg

elm-live:
  You’ve changed `src/Main.elm`. Rebuilding!

Success! Compiled 1 module.                                          
[0182] 150ms         0B GET    304 /
[0184] 562ms      407KB GET    200 /main.js
[0184] 10ms          0B GET    304 /img/bulle-reversed.svg

elm-live:
  You’ve changed `src/Main.elm`. Rebuilding!

Success! Compiled 1 module.                                          
[0186] 141ms         0B GET    304 /
[0188] 338ms      407KB GET    200 /main.js
[0188] 1ms           0B GET    304 /img/bulle-reversed.svg

elm-live:
  You’ve changed `src/Main.elm`. Rebuilding!

Success! Compiled 1 module.                                          
[0191] 243ms         0B GET    304 /
[0192] 715ms      407KB GET    200 /main.js
[0193] 1ms           0B GET    304 /img/bulle-reversed.svg

elm-live:
  You’ve changed `src/Main.elm`. Rebuilding!

Success! Compiled 1 module.                                          
[0386] 253ms         0B GET    304 /
[0388] 819ms      407KB GET    200 /main.js
[0389] 0ms           0B GET    304 /img/bulle-reversed.svg

elm-live:
  You’ve changed `src/Main.elm`. Rebuilding!

Success! Compiled 1 module.                                          
[0391] 202ms         0B GET    304 /
[0393] 828ms      407KB GET    200 /main.js
[0393] 8ms           0B GET    304 /img/bulle-reversed.svg
```

**Comment by wking-io** [comment]
This should be fixed now thanks to @lucamug and his PR on #168. Just install `elm-live@3.4.1` and let me know! :)

**Comment by sebsheep** [comment]
Seems to work ! thank you @lucamug & @wking-io !
